cprover-cpp-api/regression/call_bmc.cpp

```diff
@@ -0,0 +1,37 @@
+// Author: Fotis Koutoulakis for Diffblue Ltd.
+
+// Test file to try loading a GOTO-model into memory and running a sample verification run on it.
+#include <util/exception_utils.h>
+
+#include "api.h"
+#include "options.h"
+
+#include <iostream>
+#include <vector>
+
+int main(int argc, char *argv[])
+{
+  try
+  {
+    std::cout << "Hello from API stub" << std::endl;
+
+    // Convert argv to vector of strings for initialize_goto_model
+    std::vector<std::string> arguments(argv + 1, argv + argc);
+
+    // Create API options object, to pass to initialiser of API object.
+    auto api_options = api_optionst::create().simplify(false);
+
+    // Initialise API dependencies and global configuration in one step.
+    api_sessiont api(api_options);
+
+    // Demonstrate the loading of a goto-model from the command line arguments
+    api.load_model_from_files(arguments);
```

Comment:
Can this fail? How are error conditions handled here?

Reply:
Yes. As an example, if the filename passed doesn't exist it throws an exception.